cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -469,9 +469,13 @@ bool SubstraitToVeloxPlanValidator::validate(const ::substrait::GenerateRel& gen
     names.emplace_back(SubstraitParser::makeNodeName(inputPlanNodeId, colIdx));
   }
   auto rowType = std::make_shared<RowType>(std::move(names), std::move(types));
-
-  if (generateRel.has_generator() && !validateExpression(generateRel.generator(), rowType)) {
-    LOG_VALIDATION_MSG("Input validation fails in GenerateRel.");
+  try {
+    if (generateRel.has_generator() && !validateExpression(generateRel.generator(), rowType)) {
```

Comment:
Seems the root cause is that we don't support conversion between literal `(1, 'a')`  to RowType. Should it be better to handle this in `validateExpression` instead of catching exception here?

Reply:
Now we don't support to convert a null Struct literal to vector, and looks it's expected behavior, we have UT for this, https://github.com/apache/incubator-gluten/blob/main/backends-velox/src/test/scala/org/apache/gluten/execution/VeloxLiteralSuite.scala#L138 . But we may support that later~

I think the issue here is that the error (https://github.com/apache/incubator-gluten/blob/main/cpp/velox/substrait/SubstraitToVeloxExpr.cc#L494) is not caught properly.